README.md

```diff
@@ -75,7 +75,20 @@ EasyTranslate.detect "This is definitely English!", :confidence => true # => { :
 EasyTranslate.detect ['This is definitely English!', '我姓譚'], :confidence => true # => [{ :language => 'en', :confidence => 0.77272725 }, { :language => 'zh-CN', :confidence => 1.0 }]
 ```
 
+## NMT? PBMT? I want a neural network!
+Google Translate now replaces Phrase Based Machine Translation (PBMT) with Neural Machine Translation (NMT) automatically where possible. If you prefer PBMT or need to compare the results, you can use the `model:` parameter with either `nmt` or `base` to force the model selection:
 
+```ruby
+2.2.5 :019 > EasyTranslate.translate("El cuervo americano es un ave con plumas negras iridiscentes sobre todo su cuerpo.", from: "es", to: "en", model: "nmt")
+ => "The American Raven is a bird with iridescent black feathers over its entire body."
```

Comment:
I commonly put these inside a comment just for the syntax highlighter to be chill, but obviously optional